modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -308,13 +312,15 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
           privateShare: backupToBitgoShare.encryptedPrivateShare,
           n: backupToBitgoShare.n,
           vssProof: backupToBitgoShare.vssProof,
+          privateShareProof: backupToBitgoShare.privateShareProof,
         },
       ],
       userGPGPublicKey: userGpgKey.publicKey,
       // BitGo is the only supported third party backup as of now, so the
       // backup GPG key is the same as bitgo GPG key. Else user holds backup.
       backupGPGPublicKey: isThirdPartyBackup ? bitgoPublicGpgKey.armor() : userGpgKey.publicKey,
       enterprise: enterprise,
+      algoUsed: 'ecdsa',
```

Comment:
I dont think WP needs this parameter, what's the motivation for this?

Reply:
Hi @islamaminBitGo 
since HSM is upgraded to support uSig value soon, our clients are gonna to be on old SDK, which means they are not going to provide uSig value. But eddsa usig is already in prod.

Thus, we need to know which algo it is in order to filter in wp (not passing usig to HSM) and make it work with HSM. 
In Wp https://github.com/BitGo/bitgo-microservices/pull/27889 we will ignore the uSig value for prod ecdsa. 

Does it make sense, or am i understanding the timeline and flow in HSM and SDK wrongly?

Reply:
ah i see, gotcha